src/zrc/api/tests/test_validation.py

```diff
@@ -71,15 +71,27 @@ def test_validation_camelcase(self):
     @patch('vng_api_common.validators.fetcher')
     @patch('vng_api_common.validators.obj_has_shape', return_value=False)
     @override_settings(LINK_FETCHER='vng_api_common.mocks.link_fetcher_200')
-    def test_validate_communicatiekanaal_invalid(self, mock_has_shape, mock_fetcher):
+    def test_validate_communicatiekanaal_invalid_resource(self, mock_has_shape, mock_fetcher):
         url = reverse('zaak-list')
         body = {'communicatiekanaal': 'https://ref.tst.vng.cloud/referentielijsten/api/v1/'}
 
         response = self.client.post(url, body, **ZAAK_WRITE_KWARGS)
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         error = get_validation_errors(response, 'communicatiekanaal')
-        self.assertEqual(error['code'], ResourceValidator.code)
+        self.assertEqual(error['code'], ResourceValidator._ResourceValidator__code)
```

Comment:
Dit gaat niet goed volgens mij.

Reply:
Met de aanpassingen aan vng-api-common in [deze PR](https://github.com/VNG-Realisatie/vng-api-common/pull/29) zou dit goed moeten gaan.